engine/src/test/java/org/pentaho/di/core/row/ValueDataUtilTest.java

```diff
@@ -709,9 +762,9 @@ public void testPercent2() {
         ValueMetaInterface.TYPE_BIGNUMBER, CalculatorMetaFunction.CALC_PERCENT_2 ) );
     assertEquals( BigDecimal.valueOf( Double.valueOf( "1.96" ) ), calculate( "2", "2",
         ValueMetaInterface.TYPE_BIGNUMBER, CalculatorMetaFunction.CALC_PERCENT_2 ) );
-    assertEquals( new BigDecimal("8.00"), calculate( "10", "20",
+    assertEquals( new BigDecimal( "8.0" ), calculate( "10", "20",
```

Comment:
@lucboudreau, I changed this test result...tried to do a more "intelligent" decimal scaling